agent/util-scripts/unittests

```diff
@@ -111,14 +113,22 @@ function _run {
     echo "${tname} ${tscrpt} ${opts}"
     echo "+++ Running ${tname} ${tscrpt} ${opts}" >> ${_testout}
 
-    eval ${tscrpt} ${opts} >> ${_testout} 2>&1
+    _runout=${_testroot}/run.out
+    eval ${tscrpt} ${opts} > ${_runout} 2>&1
     sts=${?}
 
     popd > /dev/null 2>&1
     if [[ ${?} -ne 0 ]]; then
         exit 1
     fi
 
+    if [[ "${sortit}" == "sort" ]]; then
+        sort ${_runout} >> ${_testout}
+    else
+        cat ${_runout} >> ${_testout}
```

Comment:
So `${_testout}` is appended to at line 114 above, and I think it will have other output in it as well.  So if we do a `mv` at line 128 we'd lose that output, if I am reading the code correctly.